neo/Network/P2P/ProtocolHandler.cs

```diff
@@ -34,6 +34,12 @@ public ProtocolHandler(NeoSystem system)
             this.sentHashes = new FIFOSet<UInt256>(Blockchain.Singleton.MemPool.Capacity * 2);
         }
 
+        public void FlushHashes()
```

Comment:
I think it's not thread-safe to call this method directly. We should send message to the actor.